package ingenias.exception;

public class AlreadyExists extends Exception {

  public AlreadyExists() {
  }

  public AlreadyExists(String message) {
    super(message);
  }

  public AlreadyExists(String message, Throwable cause) {
    super(message, cause);
  }

  public AlreadyExists(Throwable cause) {
    super(cause);
  }
  public static void main(String[] args) {
    AlreadyExists alreadyExists1 = new AlreadyExists();
  }
}